backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/UserService.cs

```diff
@@ -37,6 +37,17 @@ public async Task<UserDto> GetCurrentUserAsync()
             return currentUserDto;
         }
 
+        public async Task<List<UserDto>> GetUsersByEmailOrNameAsync(string searchData)
+        {
+            var users = await _context.Users
+                .Where(x => x.Name.Contains(searchData) || x.Email.Contains(searchData))
```

Comment:
make this case insensitive

Reply:
It`s not case insensitive
![image](https://user-images.githubusercontent.com/71783287/189310407-fbc37b43-704d-49f3-ac08-b6ef2df507f5.png)
![image](https://user-images.githubusercontent.com/71783287/189310426-bc081ff4-57c2-4ed3-a512-28929594d4cb.png)